tests/robot-tests/tests/admin_and_public/bau/prelease_visibility.robot

```diff
@@ -0,0 +1,223 @@
+
+Force Tags  Admin  Local  Dev  AltersData
+
+Suite Setup       user signs in as bau1
+Suite Teardown    user closes the browser
+
+*** Variables ***
+${RELEASE_NAME}  Academic Year Q1 2020/21
+${RELEASE_NAME_2}  Academic Year Q1 2020/21 (not Live)
+${DATE}  Calendar Year 2000 (not Live)
+${PUBLICATION_NAME}  UI tests - public release visibility %{RUN_IDENTIFIER}
+
+*** Test Cases ***
+Create test publication and release via API
+    [Tags]  HappyPath
+    ${PUBLICATION_ID}=  user creates test publication via api   ${PUBLICATION_NAME}
+    user create test release via api  ${PUBLICATION_ID}   CY    2000
+    
+Verify release summary
+    [Tags]  HappyPath
+    user navigates to release summary from admin dashboard  ${PUBLICATION_NAME}  Calendar Year 2000 (not Live)
+    user waits until h2 is visible  Release summary
+    user checks page contains element   xpath://li/a[text()="Summary" and contains(@aria-current, 'page')]
+    user checks summary list contains  Publication title  ${PUBLICATION_NAME}
+
+Upload subject
+    [Tags]  HappyPath
+    user clicks link  Data and files
+    user waits until h2 is visible  Add data file to release
+    user enters text into element  id:dataFileUploadForm-subjectTitle   UI test subject
+    user chooses file   id:dataFileUploadForm-dataFile       ${FILES_DIR}upload-file-test.csv
+    user chooses file   id:dataFileUploadForm-metadataFile   ${FILES_DIR}upload-file-test.meta.csv
+    user clicks button  Upload data files
+
+    user waits until h2 is visible  Uploaded data files
+    user waits until page contains accordion section   UI test subject
+    user opens accordion section   UI test subject
+
+    ${section}=  user gets accordion section content element  UI test subject
+    user checks headed table body row contains  Subject title    UI test subject  ${section}
+    user checks headed table body row contains  Data file        upload-file-test.csv  ${section}
+    user checks headed table body row contains  Metadata file    upload-file-test.meta.csv  ${section}
+    user checks headed table body row contains  Number of rows   159  ${section}
+    user checks headed table body row contains  Data file size   15 Kb  ${section}
+    user checks headed table body row contains  Status           Complete  ${section}  180
+
+Go to 'Sign Off' page
+    [Tags]  HappyPath
+    user clicks link  Sign off
+    user waits for page to finish loading
+    user waits until page contains testid  public-release-url
+    ${PUBLIC_RELEASE_LINK}=  Get Text  xpath://*[@data-testid="public-release-url"]
+    Set Suite Variable  ${PUBLIC_RELEASE_LINK}
+
+Go to Public Release Link
+    [Tags]  HappyPath
+    user goes to url  %{PUBLIC_URL}
+    user goes to url  ${PUBLIC_RELEASE_LINK}
+    user waits for page to finish loading
+    # NOTE: Locally this will fail due to next showing the default dev 404 page
+    user checks page contains element  //*[@data-testid="page-title Page not found" and text()="Page not found"]
```

Comment:
I'd add another check to make sure the release page isn't showing - something like `user checks page does not contain  ${RELEASE_NAME}`

I would add this additional check in all places in this test suite where we're validating the "Page not found" page.